mnemosyne/doctor.py

```
@@ -213,6 +218,29 @@ def open_readonly_doctor_db(db_path: str | Path) -> sqlite3.Connection:
+def _load_optional_sqlite_vec(conn: sqlite3.Connection) -> bool:
+    """Load the bundled sqlite-vec module into an already read-only connection.
+
+    The Doctor must remain usable when the optional dependency is absent or
+    incompatible with the host SQLite build.  In that case, its existing
+    ``present_but_unloadable`` capability report remains the safe fallback.
+    """
+
+    try:
+        import sqlite_vec
+
+        conn.enable_load_extension(True)
```

Ruff (0.15.21)

[warning] Boolean positional value in function call

(FBT003)